.release-notes/main.md

```diff
@@ -27,8 +28,11 @@ Release notes for `TODO`.
 - Added community meetings docs
 - Added google groups in community docs and README
 - Removed all references to `TestStep` in the docs as this is not supported anymore
+- Remove TestStep reference examples
 
 ## 🎸 Misc 🎸
 
 - Added kubebuilder community recording link where chainsaw was presented
 - Added a `MAINTAINERS.md` file
+- Added makefile target to check go code coverage
```

Comment:
Remove this line, it's not relevant imho

Reply:
done